CMakeLists.txt

```diff
@@ -198,7 +198,7 @@ if(NOT "${SENTRY_UPLOAD_URL}" STREQUAL "")
 	target_compile_definitions(${PROJECT_NAME} PRIVATE SENTRY_UPLOAD="${SENTRY_UPLOAD_URL}")
 endif()
 
-target_include_directories(${PROJECT_NAME} PRIVATE core core/deps core/deps/stb core/khronos core/deps/json)
+target_include_directories(${PROJECT_NAME} PRIVATE core core/deps core/deps/stb core/khronos core/deps/json core/deps/libelf/include)
```

Comment:
Is it really necessary? This directory is already included with `target_include_directories(elf PUBLIC include)` from `core/deps/libelf/CMakeLists.txt`